controller/backing_image_controller.go

```diff
@@ -147,13 +147,13 @@ func (bic *BackingImageController) handleErr(err error, key interface{}) {
 	}
 
 	if bic.queue.NumRequeues(key) < maxRetries {
-		logrus.WithError(err).Warnf("Error syncing Longhorn backing image %v", key)
+		logrus.WithError(err).Errorf("Error syncing Longhorn backing image %v", key)
```

Comment:
Suggest keeping warn, since we handle the error by adding it to the queue for retry.

Reply:
As discussed, this is the top caller function, so we should use `Error`
Since adding to queue to retry is not longhorn mechanism bus k8s which is outside of our control